### PR TITLE
Support for Quarkus 3

### PR DESCRIPTION
**Pull request by brunobat**
Javax to jakarta packages migration

### CODE REVIEW

**Comment by ebullient** [comment]
We need to make a 2.x branch before this is integrated (and update that to continue to build against the latest Quarkus 2.x releases)

File: docs/modules/ROOT/pages/includes/attributes.adoc [hunk 1]
```diff
@@ -1,3 +1,3 @@
-:quarkus-version: 2.14.1.Final
+:quarkus-version: 3.0.0.Beta1
 :quarkus-micrometer-registry-version: 2.12.0
```
Comment (ebullient) [inline docs/modules/ROOT/pages/includes/attributes.adoc:2]: This should be a 3.x version

**Comment by brunobat** [comment]
Already created the 2.x branch from main.

**Comment by ebullient** [comment]
> Already created the 2.x branch from main.

- Dependabot in this PR needs to be updated to also maintain the 2.x branch.
- We will need to update GH Actions (in the main branch) to also build / release both branches) https://github.com/quarkiverse/quarkiverse/wiki/Migrating-to-Quarkus-3.x#set-up-github-actions-on-the-new-branch
- Quarkiverse CI needs to be updated to build both branches: https://github.com/quarkusio/quarkus-ecosystem-ci/blob/main/quarkiverse-micrometer-registry/info.yaml

**Review by ebullient — APPROVED** [review]

**Comment by brunobat** [comment]
Thanks @ebullient

**Comment by brunobat** [comment]
Included the changes to the pipeline.

**Pull request merged**